tenant-cd-api/pom.xml

```diff
@@ -25,7 +25,7 @@
             This version must match the string in all ExportPackage annotations.
```

Comment:
@jonmv Read this!

Reply:
Thanks!

Reply:
Where are the ExportPackage annotations we are talking about here?

Reply:
In the package-info.java files in tenant-cd-api (for the the junit annotations).

Reply:
Updated (although the old version exported was 5.4.2, not the expected 5.6.2)